.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,7 @@
         "dockerfile": "../Dockerfile.dev"
     },
     // Use 'forwardPorts' to make a list of ports inside the container available locally.
-    "appPort": [8080],
+    "appPort": [8080, 2002, 2003],
```

Comment:
Are these just random ports?

Reply:
port 2002 is for `soffice` and 2003 for `unoserver`